packages/bitcore-node/test/unit/models/wallet.unit.ts

```diff
@@ -1,7 +1,69 @@
-import {expect} from 'chai';
+import { expect } from 'chai';
+import * as sinon from 'sinon';
+import { WalletModel, IWalletModel } from '../../../src/models/wallet';
+import { WalletAddressModel } from '../../../src/models/walletAddress';
 
-describe('Wallet Model', function(){
-    it('should have a test which runs', function(){
-        expect(true).to.equal(true);
+describe('Wallet Model', function () {
+
+  describe('_apiTransform', () => {
+    it('should return the transform object with wallet info', () => {
+      let wallet = {
+        name: 'Wallet1',
+        singleAddress: true,
+        pubKey: 'xpub661MyMwAqRbcFa63vSTa3vmRiVWbpLWhgUsyvjfMFP7ePR5osC1rtPUkgJrB94V1YEQathfWLm9U5zaZttYPDPWhASwJGUvYvPGtofqnTGN',
+        path: 'm/44\'/0\'/0\''
+      }
+      const result = WalletModel._apiTransform(new WalletModel(wallet), { object: false });
+      const parseResult = JSON.parse(result);
+
+      expect(parseResult.name).to.be.equal(wallet.name);
```

Comment:
switch this to a deep equal and have the actual strings `Wallet1` and `xpub661MyMwAqRbcFa63vSTa3vmRiVWbpLWhgUsyvjfMFP7ePR5osC1rtPUkgJrB94V1YEQathfWLm9U5zaZttYPDPWhASwJGUvYvPGtofqnTGN`

Reply:
resolved